dashboard/app/models/new_feature_feedback.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: new_feature_feedbacks
+#
+#  id         :bigint           not null, primary key
+#  user_id    :integer          not null
+#  form_key   :string(255)      not null
```

Comment:
You could use an integer enum here instead of a string to keep the storage size down. See `SectionInstructor.status` for an example.